test/spec/Feature/Query/NullsStrip.hs

```diff
@@ -0,0 +1,80 @@
+module Feature.Query.NullsStrip where
+
+import Network.Wai (Application)
+
+import Network.HTTP.Types
+import Test.Hspec
+import Test.Hspec.Wai
+import Test.Hspec.Wai.JSON
+
+import Protolude  hiding (get)
+import SpecHelper
+
+spec :: SpecWith ((), Application)
+spec =
+  describe "Stripping null values from JSON response" $ do
+    let arrayStrip = ("Accept", "application/vnd.pgrst.array+json;nulls=stripped")
+    let singularStrip = ("Accept", "application/vnd.pgrst.object+json;nulls=stripped")
+
+    context "strip nulls from response" $ do
+      it "strips nulls when Accept: application/vnd.pgrst.array+json;nulls=stripped" $
+        request methodGet  "/organizations?select=*"
```

Comment:
Let's also add a test for `POST` and `PATCH`. You can test the result by adding `Prefer: return=representation`. Also, all writes are rolled back in the test suite ([db-tx-end=rollback](https://postgrest.org/en/stable/references/configuration.html#db-tx-end)) so you don't have to worry about the other tests breaking.